scene/resources/material.cpp

```diff
@@ -3546,7 +3546,7 @@ bool StandardMaterial3D::_set(const StringName &p_name, const Variant &p_value)
 			idx++;
 		}
 
-		WARN_PRINT("Godot 3.x SpatialMaterial remapped parameter not found: " + String(p_name));
+		WARN_PRINT("Blazium 3.x SpatialMaterial remapped parameter not found: " + String(p_name));
```

Comment:
There is no Blazium 3.x as of right now, this should probably be left as "Godot 3.x"

Reply:
fixed https://github.com/blazium-engine/blazium/pull/30/commits/a11e1f011a19a6fab53b6533e424d162f39339fb